### ISSUE TITLE
`roc run` crashes using cli-platform 

### ISSUE THREAD

**Issue by lukewilliamboswell** [roc-lang/roc#4607]
I wanted to use Roc to make a script to automate a series of find and replaces in a file. I wrote the following using cli-platform which I think should work, however it crashes. Is there something obvious I am missing here?

```
luke@192-168-1-108 tui % RUST_BACKTRACE=1 roc run fix-glue.roc 
🔨 Rebuilding platform...
thread '<unnamed>' panicked at 'not yet implemented: Report a file open error', src/lib.rs:421:13
stack backtrace:
   0: rust_begin_unwind
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/a55dd71d5fb0ec5a6a3a9e8c27b2127ba491ce52/library/core/src/panicking.rs:142:14
   2: roc_fx_fileReadBytes
             at /Users/luke/Documents/GitHub/roc/examples/cli/cli-platform/src/lib.rs:421:13
   3: _roc_fx_fileReadBytes_fastcc_wrapper
   4: _Effect_effect_closure_fileReadBytes_a962c99ae0666415e78efb96bab139f404abe9f9be88e317ee7e4c473dfaa32
   5: _Effect_effect_map_inner_ab677189bbd88d8d69abceb88946f32a36333b36d4e75cb1ae69ef3c4cf1fc2
   6: _Effect_effect_after_inner_d16a7ad8717a9e74a2a28d29899854f2faa6b889f774043e6a25b6f2ff3a7
   7: _Effect_effect_after_inner_46f83e3adcc4f4f3eecc56ca777afc551ff50c61ae15f1486e3497cd51bfb
   8: _roc__mainForHost_1__Fx_caller
   9: host::call_the_closure
             at /Users/luke/Documents/GitHub/roc/examples/cli/cli-platform/src/lib.rs:241:5
  10: rust_main
             at /Users/luke/Documents/GitHub/roc/examples/cli/cli-platform/src/lib.rs:230:9
  11: _main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
fatal runtime error: failed to initiate panic, error 5
zsh: abort      RUST_BACKTRACE=1 roc run fix-glue.roc
```

```
app "fix-glue"
    packages { pf: "/Users/luke/Documents/GitHub/roc/examples/cli/cli-platform/main.roc" }
    imports [
        pf.File,
        pf.Path.{ Path },
        pf.Task,
        pf.Stdout,
        pf.Task.{ Task },
        pf.Process,
        pf.Stderr,
    ]
    provides [main] to pf

main : Task {} []
main =
    path = Path.fromStr "src/glue.rs"

    task = 

        input <- File.readUtf8 path |> Task.await
        
        output = Str.replaceEach input "discriminant_KeyCode" "DiscriminantKeyCode"
        
        when output is 
            Ok str -> File.writeUtf8 path str
            Err _ -> Stderr.line "Error."
    
    Task.attempt task \result ->
        when result is 
            Err _ -> 
                Process.exit 1
            Ok _ -> 
                Stdout.line "Success, glue updated."
```

**Comment by ghigt** [comment]
I think you just missed to put a file name `glue.rs` in your `src/` folder :)

**Comment by lukewilliamboswell** [comment]
😃

**Comment by lukewilliamboswell** [comment]
Fixed by [basic-cli PR#4](https://github.com/roc-lang/basic-cli/pull/4)
